src/components/Logo/Logo.scss

```diff
@@ -0,0 +1,15 @@
+.Logo img {
+    width: 332px;
+    height: 194px;
+    margin-top: 20%;
+
+}
+
+@media (width:375px) and (height:667px){
```

Comment:
base ให้เป็นขนาดของ mobile แล้ว @media ใช้ @media screen and (min-width: 1280px) ให้เป็น pattern เดียวกันหมด จะมีตัวอย่างอยู่ ลองเข้าไปดูใน components/nav/Menu/Menu.scss หรือ components/layout/Containe/Container.scss

Reply:
แก้ไขแล้วครับ 👨‍💻👨‍💻👨‍💻